docs/en/operations/settings/settings.md

```diff
@@ -1230,4 +1230,14 @@ Type: URL
 
 Default value: Empty
 
+## background_pool_size {#background_pool_size}
```

Comment:
Looks like `_` should be escaped (i.e. `## background\_pool\_size {#background_pool_size}` over `## background_pool_size {#background_pool_size}`)

Reply:
No. It shouldn't be escaped. It can be escaped but our build system doesn't need it and I don't know other systems which require such escaping.

Reply:
It is done in #9763 (cc: @blinkov )
I guess because `_` used to make the text _italic_